MessengerOnFileForLinux/scripts/local_scripts/run_messenger.sh

```diff
@@ -2,7 +2,7 @@
 rm Logger_default.txt
 
```

Comment:
pamietaj o INT: w nazwie takiego commita.

Reply:
1. To było sumowane
2. INT i MSG_1 idzie dopiero od FB_4 bo jeszcze nie mamy choćby tasków ponumerowanych.